sdk/java/build.gradle

```diff
@@ -123,7 +123,7 @@ javadoc {
     if (JavaVersion.current().isJava9Compatible()) {
         options.addBooleanOption('html5', true)
     }
-    options.jFlags("-Xmx2g", "-Xms512m")
+    options.jFlags("-Xmx8g", "-Xms512m")
```

Comment:
can we rely on having that much on CI runners?

Reply:
Assume this will just act as unbounded and will fail if the host won't give it more memory - this is only the max and not the inital memory to be reserved?

Reply:
this is max, yes

Reply:
Is the extra memory required?  I'm not opposed to the change, but curious what the motivation is.

Reply:
Yes, these changes come from upstream (in the `make build`, particularly, building the Java sdk step) when we updated to the latest Java gen version. Xmx indicates the maximum allowed, so it won't fail on start, only if the program requires more and the host can't allocate it.